pm.js

```diff
@@ -692,7 +691,7 @@ function PatternsShow(nodes, parent) {
 					break
 				case PAT.INVERSESET:
 					{
-						let element = CreateDiv("inverseset", parent, "[^...]", "Inverse set.", "Matches any character except:")
+						let element = CreateDiv("inverse set", parent, "[^...]", "Inverse set.", "Matches any character except:")
```

Comment:
Any specific reason making make with space, is it for classList.contains?

Reply:
The `split` on line 586 converts that into two separate classes